images/scripts/ostree.setup

```diff
@@ -28,14 +28,19 @@ if [ "$IMAGE" == "fedora-coreos" ]; then
 fi
 
 if [ "$IMAGE" == "rhel4edge" ]; then
+    useradd -m -d /var/home/admin admin
+    usermod -a -G wheel admin
+    mkdir -p /var/home/admin/.ssh
+    cp -f /var/home/installer/.ssh/authorized_keys /var/home/admin/.ssh/authorized_keys
+    chown -R admin:admin /var/home/admin
+    restorecon -r /var/home/admin/.ssh
+
     # Setup password for admin user as edge management only sets ssh key
     echo foobar | passwd --stdin admin
 
     # Setup password for root user
     echo foobar | passwd --stdin root
 
-    # HACK: Fix .ssh ownership, see https://issues.redhat.com/browse/THEEDGE-3033
-    chown admin:admin /var/home/admin/.ssh
```

Comment:
Fedora-coreos uses the same script so is this not needed there?

Reply:
Hi @jelly I've completely changed use creation, this issue is already resolved so could be removed. 
there is some issue inside how admin user inside is created, so that I'm now trying to workaorund the issue (some tests fails with strange reasons, some change in sudo for user)